index.html

```diff
@@ -268,6 +268,41 @@ <h3 class="sub_title">
           </h3>
         </ul>
       </section>
+      <section class="contact_form">
+        <h2 class="title">Contact me</h2>
+        <p class="contact_form_des">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="form_container">
```

Comment:
- [ ] Your form looks good but it is not working properly. To work the form needs the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly read the getting started section and apply the formspree service to your form.

Reply:
@leonardodiasb, Thank you so much for your review. I will fix it quickly. Thanks once again fo remained me. 💯  I was forgot to grab the form data. 😊